examples/MQTTBlinds/config.h

```diff
@@ -34,10 +37,18 @@
 // maximum. For more devices, you should have multiple ESP32's, ensure they have
 // a different MQTT_TOPIC_PREFIX, and use the DEVICE_ALLOWLIST to control up
 // to 3 am43's per ESP.
-//
+
+#define SHADE_1 "11:22:33:44:55:66"
+#define SHADE_2 "11:22:33:44:55:66"
+#define SHADE_3 "11:22:33:44:55:66"
+#define SHADE_4 "11:22:33:44:55:66"
```

Comment:
Have to successfully connected to 4 devices? If so, you should update the comment above.

Reply:
No I haven't. I liked the idea of defining all my shades so that when I'm setting up multiple ESPs, it's a simple one line change. Removed the SHADE_4 to reduce confusion